g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/WebConfiguration.java

```diff
@@ -78,7 +78,7 @@
 	@Value("${vipservice.cross.domain.maxage}")
 	private String maxAge;
 
-	@Value("${translation.bundle.file.basepath}")
+	@Value("${translation.bundle.file.basepath:.}")
```

Comment:
Do you want to use the trinocular operator here? I think maybe need to change "translation.bundle.file.basepath?:."

Reply:
this is not use trinocular operator here, this is the spring boot default inject value